src/app/components/api/primengconfig.ts

```diff
@@ -8,6 +8,8 @@ export class PrimeNGConfig {
 
     ripple: boolean = false;
 
+    appendTo: any = "body" || HTMLElement || ElementRef || TemplateRef;
```

Comment:
Hi, I don't quite understand what to use the HtmlElement constructor for? mb only type is needed here?

```ts
appendTo: HtmlElement | ElementRef | TemplateRef | 'body' = 'body'
```

Reply:
It could be div, a "p-element" etc..

Reply:
Does not make sense, the string 'body' is always true when the HtmlElement constructor will be used?
```ts
const a = "body" || HTMLElement || ElementRef || TemplateRef;
a; // always 'body'
```